package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "babel-plugin-template-html-minifier",
 	"description": "Minify HTML in tagged template strings using html-minifier",
-	"version": "3.1.0",
+	"version": "3.0.2",
```

Comment:
Please remove this version change, this field is managed by `standard-version`.

Reply:
Hmm I didn't mean to change the version